integration-tests/kubernetes/quarkus-standard-way/src/test/java/io/quarkus/it/kubernetes/KubernetesWithMetricsTest.java

```diff
@@ -73,7 +73,7 @@ public void assertGeneratedResources() throws IOException {
                 assertThat(deploymentSpec.getTemplate()).satisfies(t -> {
                     assertThat(t.getMetadata()).satisfies(meta -> {
                         assertThat(meta.getAnnotations()).contains(entry("prometheus.io/scrape", "true"),
-                                entry("prometheus.io/path", "/met"), entry("prometheus.io/port", "9090"),
+                                entry("prometheus.io/path", "/q/metrics"), entry("prometheus.io/port", "9090"),
```

Comment:
It changed from `met` to the default of `metrics` here because I removed `quarkus.smallrye-metrics.path` for this test as I also wanted to have a test that used the default path